docs/how_to/insert.md

```diff
@@ -0,0 +1,5 @@
+# Insert Capabilities
+
+Every GPT Index data structure allows insertion.
+
```

Comment:
todo for later: code snippets and diagrams will make this pop!

Reply:
Yes! I will add a TODO